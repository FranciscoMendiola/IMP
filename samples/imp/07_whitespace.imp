x:=1;y:=2;z:=3
a  :=  10
b	:=	20
