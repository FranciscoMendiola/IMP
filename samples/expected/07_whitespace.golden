Token {tokenKind = Id, lexeme = "x", position = 0}
Token {tokenKind = Assign, lexeme = ":=", position = 1}
Token {tokenKind = IntLit, lexeme = "1", position = 3}
Token {tokenKind = Semi, lexeme = ";", position = 4}
Token {tokenKind = Id, lexeme = "y", position = 5}
Token {tokenKind = Assign, lexeme = ":=", position = 6}
Token {tokenKind = IntLit, lexeme = "2", position = 8}
Token {tokenKind = Semi, lexeme = ";", position = 9}
Token {tokenKind = Id, lexeme = "z", position = 10}
Token {tokenKind = Assign, lexeme = ":=", position = 11}
Token {tokenKind = IntLit, lexeme = "3", position = 13}
Token {tokenKind = Id, lexeme = "a", position = 15}
Token {tokenKind = Assign, lexeme = ":=", position = 18}
Token {tokenKind = IntLit, lexeme = "10", position = 22}
Token {tokenKind = Id, lexeme = "b", position = 25}
Token {tokenKind = Assign, lexeme = ":=", position = 27}
Token {tokenKind = IntLit, lexeme = "20", position = 30}
